.rustfmt.toml

```
@@ -1,8 +1,13 @@
 max_width = 90 # changed
+comment_width = 90 # changed
+wrap_comments = true # changed
 normalize_comments = true # changed
 imports_layout = "Vertical" # changed
 imports_granularity = "Crate" # changed
 trailing_semicolon = false # changed
 edition = "2021" # changed
 use_try_shorthand = true # changed
-use_field_init_shorthand = true # changed
\ No newline at end of file
+use_field_init_shorthand = true # changed
+reorder_impl_items = true # changed
+normalize_doc_attributes = true # changed
+format_macro_matchers = true # changed
```

Not looking to start a war, but may I suggest
```suggestion
format_macro_matchers = true # changed
hard_tabs = true
```

---

It makes keyboard navigation a bit easier and allows easier tuning in the IDE of spacing

---

I don't know the preferences of others, but I like four spaces more=) It looks better in GitHub than tabs